pybandits/quantitative_model.py

```
@@ -1148,3 +1148,301 @@ class CmabZoomingModelCC(BaseCmabZoomingModel, QuantitativeModelCC):
+        """
+        context = kwargs.get("context")
+        if context is None:
+            raise TypeError(
+                "_quantitative_update() for BaseQuantitativeBayesianNeuralNetwork requires 'context' in kwargs."
+            )
+        context = np.atleast_2d(context)
+        # Filter out None quantities and align rewards; flatten rewards if List[List[BinaryReward]]
+        flat_rewards: List[BinaryReward] = (
+            [r[0] for r in rewards] if rewards and isinstance(rewards[0], list) else list(rewards)
+        )
+        pairs = [(q, r) for q, r in zip(quantities, flat_rewards) if q is not None]
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)